### DISCUSSION TITLE
UserInterface

### DISCUSSION THREAD

**Discussion by sajjadarashhh** [CosmosOS/Cosmos#2265] [Ideas]
we must be make powerful UserInterface for Cosmos os and its very useful

**Comment by EnZon3** [comment]
you can read the documentation on the GUI library for Cosmos, at [](https://github.com/CosmosOS/Cosmos/wiki/Somewhat-indepth-Canvas-Changes-explained)

**Reply by sajjadarashhh** [reply]
@EnZon3 at...!?

**Reply by quajak** [reply]
https://cosmosos.github.io/articles/Kernel/CGS.html
